.gitignore

```diff
@@ -11,6 +11,10 @@
 # production
 /build
 
+# portability
+/api/__pycache__
```

Comment:
Could we remove these for the rollback? It's causing some issues when trying to merge to master, we can add back when deploying to Heroku.